cucoslib/workers/recommender.py

```diff
@@ -257,6 +258,31 @@ def get_reference_stacks_from_graph(self, list_packages):
 
         return list_ref_stacks
 
+    def get_version_information(self, input_list, ecosystem):
+        """Fetch the version information for each of the packages"""
+        input_packages = []
+        for package in input_list:
+            input_packages.append(package)
+
+        str_query = "g.V().has('ecosystem',ecosystem).has('name',within(input_packages)).as('pkg').out('has_version').hasNot('cve_ids').as('ver').select('pkg','ver').by(valueMap()).dedup()"
+        payload = {
+            'gremlin': str_query,
+            'bindings': {
+                'ecosystem': ecosystem,
+                'input_packages': input_packages
+            }
+        }
+        
+        #Query Gremlin with packages list to get their version information
+        gremlin_response = self.execute_gremlin_dsl(payload)
+        if gremlin_response is None:
+            return []
+        
```

Comment:
There are 2 blank lines between `return []` and (now removed) `print(gremlin_response)` and PEP 8 doesn't like it.